Added Extensions not showing up in Catalogues FAQ (#88)
tachiyomiorg/website: src/help/faq/extensions.md

Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
:::

### Why are my extensions not showing up in Catalogues?

> If you have freshly reinstalled Tachiyomi app, the installed extensions won't show up in `Catalogues` until you enable them. 

Reviewer: ```suggestion
If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them. 
```

Author: Made all requested changes

> ::: guide

Reviewer: ```suggestion
::: guide GUIDE
```

> If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them. 

Reviewer: ```suggestion
If you've freshly reinstalled the **Tachiyomi** app, the installed extensions might not show up
in **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)** until you enable them. 
```
Temporary fix for icon not on same-line.

Author: So the line break happens after up right? Got it. Also thanks for the dark theme it works well now

> To fix this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings_applications"/> Sources](/help/guides/catalogues/#sources)** . After that, enable the language of your extensions  and then enable the desired sources.

Reviewer: ```suggestion
To fix this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings"/> Sources](/help/guides/catalogues/#sources)**. Now you can proceed to enabling your desired sources.
```

Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
:::

### ### Why are my extensions not showing up in Catalogues?

> If you have freshly installed/reinstalled Tachiyomi app, the installed extensions won't show up in `Catalogues` until you enable them. To do this, go to **[<MaterialIcon icon-name="explore"/> Catalogues](/help/guides/catalogues)**→**[<MaterialIcon icon-name="settings_applications"/> Sources](/help/guides/catalogues/#sources)** . After that, enable the language of your extensions (such as All, English, Spanish etc) and then enable the sources you want to see in `Catalogues`.

Reviewer: Can you try following what I did more closely on the other sections?

Author: What do you want me to change?
